Students/Semester2/lessons/students/1_basics/2_vars_and_types/2R_variables_and_typestodo.ipynb

```diff
@@ -38,8 +38,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Assign an string to the variable \"mary_had_a_little_lamb\", then print it.\n",
-    "# mary_had_a_little_lamb = # FIXME: Uncomment to complete\n",
+    "# Assign an string to the variable \"Mary_had_a_little_lamb\", then print it.\n",
```

Comment:
variable shouldnt be named in sentence case

Reply:
THIS_IS_A_CONSTANT = "WONT CHANGE" # python constant
snake_case = "snake" # this is how we name python vars
camelCase = "camel" # this is usually reserved for JS in our case